lib/driver.js

```diff
@@ -203,7 +203,7 @@ class AndroidUiautomator2Driver extends BaseDriver {
   }
 
   isEmulator () {
-    return !!this.opts.avd;
+    return !!(this.opts.avd || /emulator/.test(this.opts.udid));
```

Comment:
should the comparison be case-insensitive?

Reply:
i don't think so, that string is generated by google and is always lowercase AFAIK